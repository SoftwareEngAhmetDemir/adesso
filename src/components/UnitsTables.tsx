import React from 'react'
import UnitsTablesUtil from '../utils/UnitsTablesUtil';

function UnitsTables() {
  return (
   <UnitsTablesUtil/>
  )
}

export default UnitsTables;