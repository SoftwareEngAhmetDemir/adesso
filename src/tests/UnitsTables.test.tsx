import { shallow } from 'enzyme';
import UnitsTables from '../components/UnitsTables';
import { BrowserRouter } from 'react-router-dom';

test('should test UnitsTables component', () => {
 const wrapper = shallow( <BrowserRouter><UnitsTables  /></BrowserRouter>);
 expect(wrapper).toMatchSnapshot();
//  const wrapper2 = shallow(<UnitsTables  />);
//  expect(wrapper2).toMatchSnapshot();
});