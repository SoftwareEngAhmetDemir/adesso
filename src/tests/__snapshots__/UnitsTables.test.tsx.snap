// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test UnitsTables component 1`] = `
<Router
  location={
    Object {
      "hash": "",
      "key": "default",
      "pathname": "/",
      "search": "",
      "state": null,
    }
  }
  navigationType="POP"
  navigator={
    Object {
      "action": "POP",
      "createHref": [Function],
      "createURL": [Function],
      "encodeLocation": [Function],
      "go": [Function],
      "listen": [Function],
      "location": Object {
        "hash": "",
        "key": "default",
        "pathname": "/",
        "search": "",
        "state": null,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <UnitsTables />
</Router>
`;

exports[`should test UnitsTables component 2`] = `
<Router
  location={
    Object {
      "hash": "",
      "key": "default",
      "pathname": "/",
      "search": "",
      "state": null,
    }
  }
  navigationType="POP"
  navigator={
    Object {
      "action": "POP",
      "createHref": [Function],
      "createURL": [Function],
      "encodeLocation": [Function],
      "go": [Function],
      "listen": [Function],
      "location": Object {
        "hash": "",
        "key": "default",
        "pathname": "/",
        "search": "",
        "state": null,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <UnitsTables />
</Router>
`;
